assignments/closure.js

```diff
@@ -1,20 +1,53 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+let name = "Jason";
+function returnName () {
+  return name;
+}
+console.log(name);
 
 // ==== Challenge 2: Create a counter function ====
-const counter = () => {
+const counter = function() {
   // Return a function that when invoked increments and returns a counter variable.
+  let i = 0
+return function(){
+  i = i +1
+  return i;
+} 
```

Comment:
you can also do like you do at the end of a loop i++ or even ++i. one increments before the output, the other after. don't try to over think it. sometimes you know the answer even if it seems too simple.